assets/sass/_container.scss

```diff
@@ -0,0 +1,33 @@
+.auth-container {
```

Comment:
We should scope this to #okta-sign-in, i.e.:
```css
#okta-sign-in.auth-container {
}
```

As well as the other places that we have this. This is to answer the question of "what happens if they have their own .auth-container styles?"